test/hotspot/jtreg/containers/docker/DockerBasicTest.java

```diff
@@ -81,4 +82,23 @@ private static void testHelloDocker() throws Exception {
             .shouldHaveExitValue(0)
             .shouldContain("Hello Docker");
     }
+
+
+    private static void testJavaVersionWithCgMounts() throws Exception {
+        DockerRunOptions opts =
+            new DockerRunOptions(imageNameAndTag, "/jdk/bin/java", "-version")
+            .addJavaOpts("-Xlog:os+container=trace")
+            .addDockerOpts("-v", "/sys/fs/cgroup:/cgroups-in:ro");
```

Comment:
I'd suggest, to first add a test which verifies that without `-Xlog:os+container=trace` option, `java -version` on such a container doesn't print **any** `os+container` logs (it used to print a warning in the` -v /sys/fs/cgroup:/cgroups-in:ro` case). That's all this test should verify.

Reply:
done